cves/kernel/CVE-2018-12232.yml

```
@@ -284,8 +295,10 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    There is not actually a IPC being affected here. It's just two seperate functions/functionalities
```

_an_ IPC

---

Oops! "a IPC" has been changed to "an IPC".
```suggestion
    There is not actually an IPC being affected here. It's just two seperate functions/functionalities
```